Mathlib/CategoryTheory/Category/Bipointed.lean

```diff
@@ -27,9 +27,13 @@ universe u
 
 variable {α β : Type _}
 
+set_option linter.uppercaseLean3 false
+
 /-- The category of bipointed types. -/
 structure Bipointed : Type (u + 1) where
-  pt : Type u
+  /-- the underlying type -/
```

Comment:
You don't need to add these docstrings during porting

Reply:
Aren't they needed to appease the linter?

Reply:
No, they're not

Reply:
Good. All checks passed.